.github/workflows/release-snapshot.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         # Snapshot release may only be updated for commits to the main branch.
         if: github.ref == 'refs/heads/main'
 
-        uses: softprops/action-gh-release@v1
+        uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 # v1
```

Comment:
This is correct https://github.com/softprops/action-gh-release/releases/tag/v1, but not the latest: https://github.com/softprops/action-gh-release/releases/tag/v2.0.1